controllers/cloud.redhat.com/providers/autoscaler/simple.go

```diff
@@ -0,0 +1,148 @@
+package autoscaler
+
+import (
+	"fmt"
+	"log"
+
+	res "k8s.io/apimachinery/pkg/api/resource"
+
+	crd "github.com/RedHatInsights/clowder/apis/cloud.redhat.com/v1alpha1"
+	"github.com/RedHatInsights/clowder/controllers/cloud.redhat.com/config"
+	"github.com/RedHatInsights/clowder/controllers/cloud.redhat.com/providers"
+	deployProvider "github.com/RedHatInsights/clowder/controllers/cloud.redhat.com/providers/deployment"
+	apps "k8s.io/api/apps/v1"
+	v2 "k8s.io/api/autoscaling/v2"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func ProvideSimpleAutoScaler(app *crd.ClowdApp, appConfig *config.AppConfig, sp *providers.Provider, deployment crd.Deployment) error {
+	cachedDeployment, err := getDeploymentFromCache(&deployment, app, sp)
+	if err != nil {
+		log.Println(err)
+		return err
+	}
+	deploymentHPA := makeDeployemntSimpleHPA(&deployment, app, appConfig, cachedDeployment)
+	hpaResource := deploymentHPA.getResource()
+
+	err = sp.Client.Create(sp.Ctx, &hpaResource)
```

Comment:
This appears to be using the client directly to create resource and hence it will not be tracked by Clowder, the resource cache should be used here so that Clowder can clean up the HPA if the mode is changed or teh app is deleted.